lib/routes/charge/items/item_avatar.dart

```diff
@@ -68,6 +75,46 @@ Color fromHex(String hexString) {
   return Color(int.parse(buffer.toString(), radix: 16));
 }
 
+class _IconAvatar extends StatelessWidget {
+  final double radius;
+  final String iconName;
+  final bool useDecoration;
+
+  _IconAvatar(this.radius, this.iconName, this.useDecoration);
+
+  @override
+  Widget build(BuildContext context) {
+    Map<String, IconData> iconMapping = IconMap().iconMap;
+    String iconName = this.iconName.substring(5);
```

Comment:
It is better to hide our internal protocol at this level and pass the icon name to this class.
The caller is aware of the protocol and should strip the prefix.

Reply:
[Done](https://github.com/breez/breezmobile/pull/319/commits/022093abc1a629a528afb157ec290d18e185ad4b)